hibernate-core/src/test/java/org/hibernate/orm/test/dialect/function/AltibaseFunctionsTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.orm.test.dialect.function;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.community.dialect.AltibaseDialect;
+
+import org.hibernate.testing.junit4.BaseCoreFunctionalTestCase;
+import org.hibernate.testing.orm.junit.RequiresDialect;
+import org.junit.Test;
+
+@RequiresDialect(AltibaseDialect.class)
+public class AltibaseFunctionsTest extends BaseCoreFunctionalTestCase {
```

Comment:
Please move this test and associated entities/mappings to the community dialects module. Also, for new tests, please use the JUnit 5 APIs. Take a look at e.g. `org.hibernate.orm.test.any.hbm.AnyEagerHbmTest` as an example.

Reply:
OK. I'm gonna move this class to community module and change to JUnit 5 style.